pkg/util/atomic_boolean.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
nit: [Bad package names](https://blog.golang.org/package-names#TOC_5.)

> Avoid meaningless package names. Packages named util, ... provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused.

Maybe this could be in `pkg/atomic/` instead? or

Reply:
Hm, will put this into a separate package then. :P

Reply:
Done in https://github.com/containerd/cri-containerd/pull/552